jsdoc/plugins/highcharts.jsdoc.js

```diff
@@ -1143,6 +1154,11 @@ Highcharts.chart('container', {
     }]
 });
 \`\`\`
+
+When using the tooltip options with series, it is important to understand
```

Comment:
Can we have a `**Tooltip:** ` in front of it? And move TypeScript down here as I do not think that TypeScript should be the introduction text for series options.

Reply:
I forgot to remove this text section when testing via node_modules in the main HC repo. Sorry about that. My thinking is that tooltip is not that unique - a bigger excluded prop is data IMO, so a more general description is what I am aiming at. I like the idea of moving the TS section down here so it is not in the middle of the main description 👍